_data/projects/AMPProject.yml

```diff
@@ -0,0 +1,13 @@
+name: The AMP Project (Accelerated Mobile Pages)
+desc: The AMP Project provides a framework (built on standard web technologies) to make it easy to build fast web pages.
+site: https://github.com/ampproject/amphtml/
+tags:
+- amp
+- css
+- documentation
+- html
+- javascript
+- web
+upforgrabs:
+  name: Great First Issues
+  link: https://github.com/ampproject/amphtml/milestone/25
```

Comment:
FYI, the link you've used will not render an issue count on the website - it will only provide a link. If you have a label set up for GFIs, you can use that (in the form of `https://github.com/ampproject/amphtml/labels/label%20here` or similar). 

It may end up that we could do a count of issues on a certain milestone, but that's going to require an API shift (see https://github.com/up-for-grabs/up-for-grabs.net/issues/567#issuecomment-303563550).